examples/connext_dds/basic_security/README.md

```diff
@@ -0,0 +1,57 @@
+# Example Code: Basic Security
+
+## Concept
+
+RTI Security Plugins allow you to address your security requirements in a
+granular way. It is assumed that you have a familiarity with RTI Connext for
+this example.
+
+The main aspects of security that are covered are
+
+-   Authentication - Verifying the identity of the application and/or user that
+invokes actions on DDS.
+-   Access Control - Provides a way to enforce policy decision on what DDS-related
+operations an authenticated entity can perform (ie which Domains it can join,
+which topics it can publish and subscribe to, etc)
+-   Cryptography - Implements (or interfaces with libraries that implement)
+cryptographic operations such as encryption, decryption, hashing, digital
+signatures, etc
+-   Logging - Auditing of all DDS security-related events
+
+Additionally, The Security Plugins can potentially run over any transport,
+including the builtin UDP transport with multicast and TCP transport.
+The OMG DDS Security specification addresses the security aspect of the
+communication in a one-to-many, friendly, data-centric way, enabling
+applications to define different security policies based on the nature of the
+shared data. This gives the benefits of no single point of failure, high
+performance, and high scalability
+
+In a DDS Secure system, a Governance Document defines the security requirements
+for communication. This file contains a mapping between Domain IDs and the
```

Comment:
```suggestion
for communication. At the domain level, this file contains a mapping between Domain IDs and the
```
The Governance Document also acts at the topic level, affecting endpoints from those topics ([reference](https://community.rti.com/static/documentation/connext-dds/7.1.0/doc/manuals/connext_dds_secure/users_manual/p2_core/elements_dds_secure_system.html#governance-document). I'm ok with not going over the details here, but the addition of this couple of words may help understanding that this is not the only thing the Governance Document specifies.